decidim-meetings/app/views/decidim/meetings/admin/meetings/index.html.erb

```diff
@@ -73,7 +71,9 @@
                 <%= static_map_link(meeting, {}, { class: "static-map__admin" }) unless meeting.online? %>
               </td>
             <% end %>
-            <%= td_resource_scope_for(meeting.scope) %>
+            <td>
+              <%= present(meeting).taxonomy_names.join(", ") %>
```

Comment:
On the files i have reviewed, this was presented as: 

```
<%= **model**.taxonomies.map { |taxonomy| decidim_sanitize_translated(taxonomy.name) }.join(", ") %>
```
This looks like a consistency issue ...

Reply:
well, not all follow the same consistency using presenters, I though to follow the style of the file itself in this case. If they where using a presenter, modify the presenter. Otherwise, put it directly. Maybe everything should have a presenter but this should be done in a different PR.

Reply:
IMO, when adding something, we should follow the style of the functionality (implementation), otherwise, when following the "style of the file", we may actually make it very hard to observe repetitive code that we could extract in a class at a later date. 

Just stating my oppinion here, nothing else to do in this particular case.